Libraries/Components/WebView/WebView.ios.js

```diff
@@ -462,6 +462,29 @@ var WebView = React.createClass({
   },
 
   /**
+   * Evaluate JavaScript on the current page.
+   */
+  evaluateJavaScript: async function(
+    script: string,
+    callback?: ?(error: ?Error, result: ?string) => void,
```

Comment:
OK -- I was following the convention I saw elsewhere in the codebase (and carried over from #1191) and didn't know whether new APIs should still be both callback- and promise-compatible. If that's the case, I'll drop it. Thanks again for the feedback!